### ISSUE TITLE
Number arb causes a NaN if summing values together.

### ISSUE THREAD

**Issue by SimonRichardson** [puffnfresh/bilby.js#7]
If calling arb for Number and summing multiple versions causes values to be NaN.

The question is, is this even an issue, should the arb Number method be aware of these issues, or is it just a eventuality that calling arb 5 times and then summing those together causes a NaN. 

I was running into this when concating Tuples together with numbers as the values and hitting NaN.

**Comment by puffnfresh** [comment]
Yes, it's really annoying. I guess there should be a combinator for `arb` to filter based on a predicate.
